DataScience/FeatureImportance.py

```diff
@@ -115,7 +115,12 @@ def get_feature_importance(log_file, ml_args, warmstart_model=None, min_num_feat
 
     vw_inv_hash_cmd = vw_base + ' --invert_hash {0}'.format(invHash_fp)
     Logger.info('command to get invert hash file: {0}'.format(vw_inv_hash_cmd))
-    os.system(vw_inv_hash_cmd)
+    invert_hash_process = Popen(vw_inv_hash_cmd.split())
+    try:
+        invert_hash_process.wait(invert_hash_timeout)
+    except TimeoutExpired:
+        # Since the invert hash file is not generated, subsequent commands will fail as well so the feature importance computation stops early.
+        invert_hash_process.kill()
```

Comment:
can we:
1) Explicitly log that process is stopped due to timeout
2) Explicitly return some empty (or "invalid") result in this case? I.e. [[],[]]

Reply:
done, thanks!